pkg/gather/gather.go

```diff
@@ -193,32 +190,17 @@ func recordGatheringFunctionResult(
 	}, allErrors
 }
 
-func readMemoryUsage() (int, error) {
-	b, err := os.ReadFile("/sys/fs/cgroup/memory/memory.usage_in_bytes")
```

Comment:
This was just printing the error in the log because this file is not available in the container anymore. I can create a followup spike.